src/graphnet/data/constants.py

```diff
@@ -30,6 +30,7 @@ class FEATURES:
         "t",
     ]
     KAGGLE = ["x", "y", "z", "time", "charge", "auxiliary"]
+    LIQUIDO = ["sipmID", "sipm_x", "sipm_y", "sipm_z", "t", "var"]
```

Comment:
Would it not be better to keep x,y,z,t as the first entries for consistency?

Reply:
Hey @Aske-Rosted! I would ordinarily agree, but `x` is a reserved name in our data representations - i.e. `data.x` gives us the entire input to models (see [here](https://github.com/graphnet-team/graphnet/blob/16bf0cede70f5575762d2e969ced1fb10ba9aa5b/src/graphnet/models/graphs/graph_definition.py#L435)). So it is actually a bit unfortunate that the naming in the competition used a generic xyzt naming scheme. Also, LiquidO does not have PMTs like our usual experiments do; they use scintillating fibers arranged in a grid-like structure, and the xyz currently represents the position of the silicon photomultiplier (SiPM) that read out the signal from a given fiber (one attached to each end), which make out points on "the boundary" of the detector and not inside the detector volume. I think adding `sipm_` is a fair way to make this distinction clearer.
![image](https://github.com/graphnet-team/graphnet/assets/48880272/6a1dce74-2e47-41af-b100-ef69146c683f)
(image from [here](https://neutrinos.portales.ciemat.es/liquido))

Reply:
Sorry I was a bit unclear I don't think there is any problem with actual names, bur thought that the order i.e having it be `LIQUIDO = [ "sipm_x", "sipm_y", "sipm_z","sipmID", "t", "var"]`
might be slightly more standard, although it is a very minor thing. I'll approve the PR and leave this at your discretion.

Reply:
Sorry I misunderstood. I changed the order!